### PR TITLE
fix issue 15336, opIn undocumented for JSONValue

### PR DESCRIPTION
**Pull request by BBasile**
+ fix, other ddocs, use standard section to indicate what's thrown

### CODE REVIEW

**Comment by yebblies** [comment]
According to grep, `Throws: XXXException ...` is the prevailing phobos style, not `Throws: a XXXException`.

Your commit message should be something like `Issue 15336 - opIn undocumented for JSONValue`

The rest looks good.

**Comment by BBasile** [comment]
PR updated, dont use anymore throw ddoc section.

File: std/json.d [hunk 1]
```diff
@@ -166,7 +165,8 @@ struct JSONValue
     }
 
     /// Value getter/setter for $(D JSON_TYPE.STRING).
-    /// Throws $(D JSONException) for read access if $(D type) is not $(D JSON_TYPE.STRING).
+    /// Throws $(D JSONException) for read access if $(D type) is not
```
Comment (Hackerpilot) [inline std/json.d]: All of these should be `Throws:` instead of just `Throws`. The colon on the end lets DDoc know that this is a separate section.

Comment (Hackerpilot) [inline std/json.d]: I think what @yebblies meant is that the word `a` should be removed, not that you should not use the ddoc `Throws` section.

Comment (yebblies) [inline std/json.d]: Yeah, that's what I meant.  Just a consistency change, functionally it was fine.

Comment (BBasile) [inline std/json.d]: Ok updated again. This should really be fine now.

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**